AdvanceTaskMarsPart1/Pages/NotificationPage.cs

```diff
@@ -0,0 +1,105 @@
+        private IWebElement ShowLessButton    => driver.FindElement(By.XPath("//a[text()='...Show Less']"));
+        private IWebElement SelectCheckbox    => driver.FindElement(By.XPath("(//div[@class='one wide column']/input[@type='checkbox'])[1]"));
+        private IWebElement DeleteButton      => driver.FindElement(By.XPath("//div[@data-tooltip='Delete selection']"));
+        private IWebElement Message           => driver.FindElement(By.XPath("//div[@class='ns-box-inner']"));
+        private IWebElement MarkSelectionAsRead => driver.FindElement(By.XPath("//i[@class='check square icon']"));
+        private IWebElement SelectAllButton     => driver.FindElement(By.XPath("//div[@data-tooltip='Select all']"));
+        private IWebElement UnSelectAllButton   => driver.FindElement(By.XPath("//i[@class='ban icon']"));
+        private IReadOnlyCollection<IWebElement> CheckBoxSelected => driver.FindElements(By.XPath("//input[@type='checkbox']"));
+
+        public void LoadMore_Notification()
+        {
+            Wait.WaitToBeClickable("XPath", "//div[@class='ui top left pointing dropdown item']", 4);
+            NotificationTab.Click();
+            Wait.WaitToBeClickable("XPath", "//a[text()='See All...']", 4);
+            SeeAllLink.Click();
+            Wait.WaitToBeClickable("XPath", "//a[@class='ui button']", 4);
+            LoadMoreButton.Click();
+        }
+
+        public List<IWebElement> getRequestsList()
+        {
+            Thread.Sleep(4000);
+            return new List<IWebElement>(RequestsList);
+        }
+
+        public void ShowLess_Notification()
+        {
+            Wait.WaitToBeClickable("XPath", "//div[@class='ui top left pointing dropdown item']", 4);
+            NotificationTab.Click();
+            Wait.WaitToBeClickable("XPath", "//a[text()='See All...']", 4);
+            SeeAllLink.Click();
+            Wait.WaitToBeClickable("XPath", "//a[@class='ui button']", 4);
+            LoadMoreButton.Click();
+            Wait.WaitToBeClickable("XPath", "//a[text()='...Show Less']", 4);
+            ShowLessButton.Click();
+        }
+
+        public void Delete_Notification()
+        {
+            Wait.WaitToBeClickable("XPath", "//div[@class='ui top left pointing dropdown item']", 4);
+            NotificationTab.Click();
+            Wait.WaitToBeClickable("XPath", "//a[text()='See All...']", 4);
+            SeeAllLink.Click();
+            Wait.WaitToBeClickable("XPath", "(//div[@class='one wide column']/input[@type='checkbox'])[1]", 4);            
+            SelectCheckbox.Click();            
+            DeleteButton.Click();
+        }
+
+        public string getMessage()
+        {
+            Wait.WaitToExist("XPath", "//div[@class='ns-box-inner']", 4);
+            return Message.Text;
+        }
+
+        public void MarkAsRead_Notification()
+        {
+            Wait.WaitToBeClickable("XPath", "//div[@class='ui top left pointing dropdown item']", 4);
```

Comment:
There is common code for the elements you are waiting for please update this so that you are not adding the same code in the other methods!